SWEgit/src/com/company/Main.java

```diff
@@ -198,4 +199,24 @@ public static void GetAverage(String[] arr) {
         System.out.println("Average : "+sum/arr.length);
         }
     }
+ public static void shuffle(String[] arr) {
+        if (!Character.isDigit(arr[0].charAt(0))) {
+            System.out.println("Sorry the shuffle function run with array of Integers only ");
+            return;
+        } else {
+            int name_num_of_element = arr.length;
+            for (int i = 0; i < name_num_of_element; i++) {
+                int s = i + (int) (Math.random() * (name_num_of_element -i));
+                String tmp = arr[s];
+                arr[s] = arr[i];
+                arr[i] = tmp;
+
+            }
+        }
+          System.out.println("shuffle array : " );
```

Comment:
System.out.print("shuffle array :  " ) ;   ... please make the output like this ^^